src/usdb_syncer/song_loader.py

```diff
@@ -48,10 +48,17 @@ class Locations:
 
     @classmethod
     def new(
-        cls, song_id: SongId, song_dir: Path, meta_path: Path | None, headers: Headers
+        cls,
+        song_id: SongId,
+        song_check: bool,
+        song_dir: Path,
+        meta_path: Path | None,
+        headers: Headers,
     ) -> Locations:
         filename_stem = sanitize_filename(headers.artist_title_str())
         if not meta_path:
+            if song_check:
+                song_dir = song_dir.joinpath("[SC]-Songs")
```

Comment:
Maybe we should delay this until #16? I personally prefer less subfolders.

Reply:
Sure, maybe also related to #31.